### DISCUSSION TITLE
博士模板中英文摘要页的一些格式问题

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#166] [Q&A] [answered]
在中文摘要页中，将 `xx级博士生` 与 `姓名：xx` 连接到一起了，应该在 `级博士生` 和 `姓名：` 之间至少保留一个空格作为区分 
<img width="692" alt="截屏2022-10-31 17 00 03" src="https://user-images.githubusercontent.com/30745407/198970974-7397ba9d-215d-40d6-a7c4-be8ddf02a588.png">

对于英文摘要，应该也在填写的信息下方有下划线（图片特地保留一点笔画以显示确实没有给出下划线）并且填写的信息并未居中显示。
<img width="785" alt="截屏2022-10-31 16 54 27" src="https://user-images.githubusercontent.com/30745407/198970988-7fc621b7-d949-4ddc-888e-bef007479856.png">

希望：
- 中文摘要页 `级博士生` 与 `姓名：`之间保留一个空格
- 为英文摘要页的信息增加下划线以及居中显示（居中显示可选，查看学长学姐的论文有的居中有的不居中，没有硬性规定）

**Comment by atxy-blip** [comment, marked as answer]
> 至少保留一个空格作为区分

研究生院给的材料包里面的摘要模板没有这个空格，见 <https://grawww.nju.edu.cn/1076/list.htm>。如果实在要改，目前 v1.1.0 的办法是在导言区加
```tex
\ExplSyntaxOn
\njusetname{author}[a]{\g__nju_info_type_tl 生 \quad{} 姓名}
\ExplSyntaxOff
```
> 增加下划线以及居中显示

这个来来回回改了好几次，带不带下划线、是否居中都有人要求。还是单独做个选项吧。

**Reply by dreistein543** [reply]
> 研究生院给的材料包里面的摘要模板没有这个空格，见 https://grawww.nju.edu.cn/1076/list.htm。

嗯，仔细想了下，如果不加空格区分，实质上是认为 “xx专业xx级博士生”都是 “姓名”的前缀修饰
说明其实摘要页本身需要的只是“题目”“学生姓名”“导师姓名”，这样考虑确实不需要单独修改。

>增加下划线以及居中显示
>>这个来来回回改了好几次，带不带下划线、是否居中都有人要求。还是单独做个选项吧。

十分感谢您的工作

**Reply by atxy-blip** [reply]
就下划线居中吧，不加这个选项了。见 <https://github.com/nju-lug/NJUThesis/commit/9d0d5aad4fc96f412f9421c4b6586321429fe43b>。
